Resources/Maps/_Mono/Shuttles/Nfsd/anura.yml

```
@@ -542,7 +542,20 @@
     - type: RadiationGridResistance
     - type: BecomesStation
       id: Anura
-  - uid: 333
+  - uid: 2
+    components:
+    - type: MetaData
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)